src/community/label_propagation.jl

```diff
@@ -9,7 +9,7 @@ the second is the convergence history for each node. Will return after
 ### References
 - [Raghavan et al.](http://arxiv.org/abs/0709.2938)
 """
-function label_propagation(g::AbstractGraph{T}, maxiter=1000) where T
+function label_propagation(g::AbstractGraph{T}, maxiter=1000; rng::AbstractRNG=GLOBAL_RNG) where T
```

Comment:
After a quick look at the codebase, the rng seems to always be passed as an Integer seed, and then is converted to rng using `getRNG` in utils.jl. It seems it would be inconsistent to do something else. Maybe a breaking change for 2.0 ?

Reply:
Breaking no, inconsistent maybe. But I think there already was a conversation a while ago, about changing everything to `rng` for random functions. What we could do is to allow either of `seed` or `rng` as keyword argument and just emit a deprecation warning when someone uses `seed`.

Definitely would not wait for a 2.0 version with such a change.

Reply:
Ok, the deprecation road seems to be the solution.

Reply:
Found [this](https://github.com/sbromberger/LightGraphs.jl/pull/1324) and [this](https://github.com/sbromberger/LightGraphs.jl/pull/1328/files). Another stuff that was in the 2.0 dev and never made it to master. I will try to bring the remaining commits of this branch when I have the time, there was also interesting stuff about `astar`.

Reply:
The changes in these PRs are breaking though (which would not be such a big issue for 2.0), I will write something non-breaking.

Reply:
I have added a `rng_from_rng_or_seed`  helper function. While it is strictly not necessary in the case here, it serves as an example, and we can use it for other randomized functions in some further PR.